addons/web/static/src/core/utils/ui.js

```diff
@@ -227,3 +227,349 @@ export function addLoadingEffect(btnEl) {
         loaderEl.remove();
     };
 }
+
+/**
+ * jQuery find function behavior is::
+ *
+ *      $('A').find('A B') <=> $('A A B')
+ *
+ * The searches behavior to find options' DOM needs to be::
+ *
+ *      $('A').find('A B') <=> $('A B')
+ *
+ * This is what this function does.
+ *
+ * @param {jQuery} $from - the jQuery element(s) from which to search
+ * @param {string} selector - the CSS selector to match
+ * @param {boolean} [addBack=false] - whether or not the $from element
+ *                                  should be considered in the results
+ * @returns {jQuery}
+ */
+
+// TODO: Need to convert cssFind from jquery to vanilla js
+
+export function cssFind($from, selector, addBack) {
```

Comment:
I would leave this function in legacy/dom.js as it uses jquery.

Reply:
Hello @aab-odoo ,
Changes have been made, Please check it.
Thanks!!